tests/v2/test_wakunode.nim

```diff
@@ -478,7 +478,7 @@ procSuite "WakuNode":
       # invalid IP address
       discard parseRemotePeerInfo("/ip4/127.0.0.0.1/tcp/60002/p2p/16Uuu2HBmAcHvhLqQKwSSbX6BG5JLWUDRcaLVrehUVqpw7fz1hbYc")
     
-    expect ValueError:
+    expect LPError:
```

Comment:
Can ValueError no longer happen or was this wrong?

Reply:
Our implementation for check has changed and now we don't check the same and libp2p handles the scenario like other scenarios.